meeting-notes/2019/Q2/2019-06-03--js-core-dev-team-weekly.md

```diff
@@ -0,0 +1,213 @@
+     - Resulted in creating [libp2p-switch#339](https://github.com/libp2p/js-libp2p-switch/issues/339). We should handle dial backoff (blacklisting) by multiaddr instead of by peer as addresses can change over time.
+ - Blocked:
+ - Next: 
+   - More prep for IPFS Camp
+   - Finish Draft of AutoNAT spec and open PR
+   - Possibly continue with AutoNAT work in JS (time permitting)
+
+@vasco-santos
+- Done:
+  - enable browser tests on all pubsub modules
+    - [ChainSafe/gossipsub-js#28](https://github.com/ChainSafe/gossipsub-js/pull/28)
+    - [libp2p/js-libp2p-pubsub#19](https://github.com/libp2p/js-libp2p-pubsub/pull/19)
+    - [libp2p/js-libp2p-floodsub#80](https://github.com/libp2p/js-libp2p-floodsub/pull/80)
+  - update floodsub benchmarks
+    - [libp2p/js-libp2p-floodsub#82](https://github.com/libp2p/js-libp2p-floodsub/pull/82)
+  - update `libp2p` interop with new `go-libp2p`
+    - [libp2p/npm-go-libp2p-dep#3](https://github.com/libp2p/npm-go-libp2p-dep/pull/3)
+    - [libp2p/npm-go-libp2p-dep#6](https://github.com/libp2p/npm-go-libp2p-dep/pull/6)
+    - [libp2p/interop#18](https://github.com/libp2p/interop/pull/18)
+- Blocked:
+  - IPNS test [ipfs/interop#71](https://github.com/ipfs/interop/issues/71)
+- Next:
+  - add interop tests for hyprid pubsub network (needs to be implemented in gossipsub)
+    - [libp2p/interop#17](https://github.com/libp2p/interop/pull/17)
+  - integrate `interface-connection` proposal
+    - [libp2p/js-libp2p-tcp#102](https://github.com/libp2p/js-libp2p-tcp/pull/102)
+    - [libp2p/js-libp2p-websockets#82](https://github.com/libp2p/js-libp2p-websockets/pull/82)
+ - more async iterators work
+ 
+@vmx
+ - Done:
+   - https://github.com/vmx/ipld-graphviz Supports [Parasol](http://parasol.alexgirard.com/) JSON output now
+   - multiformats cleanup:
+     - I try to archive multistream: https://github.com/multiformats/multistream/issues/9
+     - Try to make multicodec README clearer: https://github.com/multiformats/multicodec/pull/134
+ - Blocked: Nope
+ - Next:
+   - More work on [STAC](https://stacspec.org/) on IPLD
+   - Playing a bit with WASM in regards to IPLD
+
+@jimpick
+ - Done:
+   - More IPFS Camp Course C - "Apps"
+   - Got HTTP Signed Exchanges "publisher" demo working (SXG Demo):
+     https://github.com/ipfs/integration-mini-projects/issues/3
+ - Blocked:
+   - Need to figure out ipfs-cluster issues for peer-base-pinner
+ - Next:
+   - More IPFS Camp
+   - Switch over to Testlab - got some ideas for content discovery tests
+   - investigate some possible bugs revealed by SXG demo
+
+
+@hugomrdias
+ - Done:
+   - https://github.com/ipfs/js-ipfs-unixfs-importer/pull/31 feat: use a rabin chunker in wasm
+   - https://github.com/ipfs/js-ipfs/pull/2110 Prepare for new aegir version
+   - https://github.com/ipfs/js-ipfs-http-client/pull/1021 Prepare for new aegir version
+   - IPFS Camps Sessions
+   - Dns resolver for IPNS
+ - Blocked:
+ - Next:
+   - Continue DNS resolver
```

Comment:
@hugomrdias do you think this will be ready in time for camp?

Reply:
//cc @Stebalien who is interested in following this work.

@hugomrdias do you have a doc that describes how this is getting implemented?